pkgs/development/python-modules/xontrib-whole-word-jumping/default.nix

```
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
+  lib,
+  fetchFromGitHub,
+
+  pytest,
+  xonsh,
+}:
+
+buildPythonPackage rec {
+  name = "xontrib-whole-word-jumping";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "xonsh";
+    repo = name;
+    rev = version;
+    sha256 = "sha256-zLAOGW9prjYDQBDITFNMggn4X1JTyAnVdjkBOH9gXPs=";
+  };
+
+  patches = [
+    ./remove-xonsh-dependency.patch
+  ];
+
+  checkInputs = [
+    pytest
+    xonsh
+  ];
+
+  meta = with lib; {
+    description = "Additional keyboard navigation for interactive xonsh shells";
+    homepage = "https://github.com/xonsh/${name}";
```

```suggestion
    homepage = "https://github.com/xonsh/xontrib-whole-word-jumping";
```
please use clickable links